awx/sso/middleware.py

```diff
@@ -20,9 +18,12 @@
 
 class SocialAuthMiddleware(SocialAuthExceptionMiddleware):
 
-    def process_view(self, request, callback, callback_args, callback_kwargs):
-        if request.path.startswith('/sso/login/'):
-            request.session['social_auth_last_backend'] = callback_kwargs['backend']
+    def __init__(self, get_response):
```

Comment:
Since we know now that `SocialAuthExceptionMiddleware` subclasses `MiddlewareMixin`, we don't need either of the `__init__` or `__call__` methods here, since we aren't doing any custom logic.

Reply:
Removed!